src/app/demos/phone-field/phone-field-demo.component.html

```diff
@@ -23,7 +23,7 @@
 <h2
   class="sky-section-heading"
 >
-  Reactive phone field with default country
+  Reactive phone field with a different default country
```

Comment:
Not sure on the reason for the change here? The other two demos don't specify a default country. Is this due to the fact that the "default" default country is the US? If so, I think maybe should change this to something like "Reactive phone field with a specified default country" or something different. I don't love the "different default' as if I didn't read the docs for the property above (or just missed the US part) then I wouldn't understand what is different about it.

Reply:
Yeah, my concern was that the first field was using the default-default country. So labeling this one as having a default country seemed like a weird distinction. I'm not in love with "different default" either. I'll change "different" to "specified."

Reply:
Actually, I changed this to "Reactive phone field with a default country specified." Hopefully that works for you.